run-tests-with-build-kogito-runtimes/pom.xml

```diff
@@ -48,96 +48,40 @@
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-decisions/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-integration-tests/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-predictions/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-processes/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-rules/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-sample/kogito-codegen-sample-generator/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-codegen-modules/kogito-codegen-sample/kogito-codegen-sample-runtime/pom.xml</pomInclude>
-
-            <!-- kogito-quarkus-parent -->
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-decisions-extension/kogito-quarkus-decisions-integration-test/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-decisions-extension/kogito-quarkus-decisions-integration-test-hot-reload/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-extension-common/kogito-quarkus-common-deployment/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-extension/kogito-quarkus-deployment/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-extension/kogito-quarkus-integration-test/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-extension/kogito-quarkus-integration-test-hot-reload/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-predictions-extension/kogito-quarkus-predictions-integration-test/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-rules-extension/kogito-quarkus-rules-integration-test/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-rules-extension/kogito-quarkus-rules-integration-test-hot-reload/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-rules-extension/kogito-quarkus-rules-integration-test-legacy/pom.xml</pomInclude>
-            <pomInclude>kogito-runtimes*/kogito-quarkus-parent/kogito-quarkus-rules-extension/kogito-quarkus-rules-integration-test-session/pom.xml</pomInclude>
-
-            <!-- kogito-workitems -->
-            <pomInclude>kogito-runtimes*/kogito-workitems/pom.xml</pomInclude>
+            <!-- includes everything, filtering is done in resolve-includes.groovy script
+            which adds invoker-run.groovy script containing 'return false' near each pom.xml
+            that should be excluded.-->
```

Comment:
The readme is now out of date for run-tests-with-build/kogito-runtimes submodule

Reply:
I've corrected that statement and added to top-level readme details about the approach and implementation. Please have a look, committing in a bit.